packages/schema/parse/src/comparison/compareSets.ts

```diff
@@ -0,0 +1,24 @@
+export enum SetComparisonType {
```

Comment:
I think SetComparisonResult is a better name
Type implies a type of comparison...